### PR TITLE
get email from ~/.gitconfig

### PR DESCRIPTION
**Pull request by nalbion**
If users have their git user set up with an email address, this can be used as the default email.

### CODE REVIEW

File: pilot/utils/arguments.py [hunk 1]
```diff
@@ -36,12 +39,10 @@ def get_arguments():
         arguments['app_id'] = str(uuid.uuid4())
 
     if 'user_id' not in arguments:
-        arguments['user_id'] = str(uuid.uuid4())
+        arguments['user_id'] = getpass.getuser()
```
Comment (LeonOstrez) [inline pilot/utils/arguments.py:42]: since https://github.com/Pythagora-io/gpt-pilot/pull/57 is merged we dont need this, right?

**Review by LeonOstrez — CHANGES_REQUESTED** [review]
can you please resolve conflicts and i can merge it then. Thank you!

File: pilot/utils/arguments.py (hunk 1 above)
Comment (nalbion) [reply pilot/utils/arguments.py:42]: yeah, it must have leaked in from the other branch

**Comment by nalbion** [comment]
Ready for merge

**Pull request merged**